Source/Renderer/ShaderSource.js

```diff
@@ -330,28 +332,33 @@ define([
 
     ShaderSource.createPickVertexShaderSource = function(vertexShaderSource) {
         var renamedVS = ShaderSource.replaceMain(vertexShaderSource, 'czm_old_main');
-        var pickMain = 'attribute vec4 pickColor; \n' +
-            'varying vec4 czm_pickColor; \n' +
-            'void main() \n' +
-            '{ \n' +
-            '    czm_old_main(); \n' +
-            '    czm_pickColor = pickColor; \n' +
-            '}';
-
+        var pickMain = 
+//language=glsl            
+'\
+attribute vec4 pickColor;\n\
+varying vec4 czm_pickColor;\n\
+void main()\n\
+{\n\
+    czm_old_main();\n\
+    czm_pickColor = pickColor;\n\
+}';
         return renamedVS + '\n' + pickMain;
     };
 
     ShaderSource.createPickFragmentShaderSource = function(fragmentShaderSource, pickColorQualifier) {
         var renamedFS = ShaderSource.replaceMain(fragmentShaderSource, 'czm_old_main');
-        var pickMain = pickColorQualifier + ' vec4 czm_pickColor; \n' +
-            'void main() \n' +
-            '{ \n' +
-            '    czm_old_main(); \n' +
-            '    if (gl_FragColor.a == 0.0) { \n' +
-            '       discard; \n' +
-            '    } \n' +
-            '    gl_FragColor = czm_pickColor; \n' +
-            '}';
+        var pickMain = pickColorQualifier + 
+//language=glsl                       
+' \
```

Comment:
It is actually supposed to be there. That was the missing space referred to in my last commit. If you look in the original code it is there as well.